sql/postgres/migrate.go

```diff
@@ -787,6 +787,11 @@ func (s *state) addIndexes(t *schema.Table, indexes ...*schema.Index) {
 			b.P("UNIQUE")
 		}
 		b.P("INDEX")
+		if c := (Concurrently{}); sqlx.Has(idx.Attrs, &c) {
+			if c.Concurrently {
+				b.P("CONCURRENTLY")
+			}
+		}
```

Comment:
```suggestion
		if c := (Concurrently{}); sqlx.Has(idx.Attrs, &c) {
			b.P("CONCURRENTLY")
		}
```
See my comment above.